_posts/2019-12-25-technical-debt.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: "Technical Debt: The First 50 years"
+date: 2020-07-01
+tags: debt devops dora 
+author: Matt Tyler
+image: 
+---
+
+'Technical debt' is a term used by engineers to describe a large number of problems that may be happening during development and operation of software. It can be difficult to get support to address technical debt due to a luck of trust between engineering functions and the wider business, and the failure of engineering teams to measure and communicate technical debt in specific terms. Technical Debt itself can't be measured but as most engineers will agree that it has an impact on the ability to both deliver and operate software, the four-key metrics presented in the DORA report can act as a suitable alternative. It is easier to argue for investment in specific areas if they can be shown to have an impact on a measure that the business understands to be linked to performance and therefore the profitability of the business.
```

Comment:
I think the opening is a bit weak. 

How about something like : _'Technical debt' is a term used by engineers to describe the general friction they encounter in working with software. It stands as a signpost for all the bad design decisions or compromises that previous engineers have made in order to accomplish what they need to achieve._

But also this sentence is wonky : _Technical Debt itself can't be measured but as most engineers will agree that it has an impact on the ability to both deliver and operate software, the four-key metrics presented in the DORA report can act as a suitable alternative._ 

should be _Technical Debt itself can't be measured but most engineers will agree that it has an impact on the ability to both deliver and operate software; the four-key metrics presented in the DORA report can act as a suitable alternative._

Reply:
The first three paragraphs were added after I wrote most of the others - to serve as an 'abstract', hence why the first three are all rather passive. I'll see if can re-jig it a little bit...